test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -169,90 +182,103 @@ public void actionPerformed(ActionEvent event) {
 
         setImageButton5.addActionListener(new ActionListener() {
             public void actionPerformed(ActionEvent event) {
-                List<BufferedImage> list = new ArrayList<>();
+                List<Image> list = new ArrayList<>();
```

Comment:
```suggestion
                List<Image> list = new ArrayList<>(4);
```
Micro-optimization: we know the number of images, the array in `ArrayList` could be sized accordingly.